### DISCUSSION TITLE
Arbitrum governor limit

### DISCUSSION THREAD

**Discussion by mizdler** [wormhole-foundation/wormhole#3258] [Governance]
20K per tx for Arbitrum is too low and we have users hitting this limit.
I suggest increasing the limit to 200K/2M for Arbitrum.

**Comment by tbjump** [comment]
I'll close this in favor of the recently raised https://github.com/wormhole-foundation/wormhole/discussions/3259, which includes the proposal to increase the Arbitrum parameters to 1M/100k. If you believe that is insufficient, please comment in that discussion.
